relay-server/src/services/buffer/envelope_store/sqlite.rs

```diff
@@ -180,6 +180,14 @@ impl DiskUsage {
     }
 }
 
+/// Enum representing the order in which [`Envelope`]s are fetched or deleted from the
+/// database.
+#[derive(Debug, Copy, Clone)]
+pub enum EnvelopesOrder {
+    MostRecent,
+    Oldest,
```

Comment:
Nit: I suggest naming the variants something like
```suggestion
    NewestFirst,
    OldestFirst,
```